### DISCUSSION TITLE
How to change the default layout?

### DISCUSSION THREAD

**Discussion by robinlej** [AnySoftKeyboard/AnySoftKeyboard#3234] [Q&A] [answered]
Hi,

I've been using ASK for more than 6 months now and it mainly works really well although I remember having trouble configuring it at first. However, I had to reinstall it today and I find myself stuck in that same time-consuming configuration step again.

Long story short: where can I switch from the 1st layout to the second?
![anysoftkeyboard-layout](https://user-images.githubusercontent.com/95914584/152682633-e2b8aa9e-c660-482e-8077-2a37f64aa999.jpg)

[Edit: OMG, I found it, I'm sorry, I must be dumb today...]

Thanks!

**Comment by menny** [comment, marked as answer]
A bit hidden, but this is the way: go to the layouts page, and long press the keyboard you want to be first. Drag it to the first position in the page

**Reply by robinlej** [reply]
Thank you. I didn't know that, and this was not my question, but it's useful to know!
My question was about the layout of the bottom line (sorry, I wasn't that clear). I finally found it under Interface > Tips & more > Bottom row (or something like that, I'm using it in French, so the translation might not be exactly that).

I think what threw my off was that there's a "Tips & more" menu both in the languages tab and in the interface tab. And since it's not explicit as to what you can find in there, I must have overlooked it thinking I had checked it already.
But it isn't that hidden either, I should have found it earlier.

**Reply by menny** [reply]
Ah... yes, in English it is called "Tweaks"

A revamp of the settings app is much required
